src/java/org/apache/cassandra/index/sai/utils/PrimaryKey.java

```diff
@@ -61,11 +83,15 @@ public PrimaryKey createTokenOnly(Token token)
             return new TokenOnlyPrimaryKey(token);
         }
 
+        /**
+         * Create a {@link PrimaryKey} for tables without clustering columns
+         */
         public PrimaryKey createPartitionKeyOnly(DecoratedKey partitionKey)
```

Comment:
nit: For `createTokenOnly()` and `createPartitionKeyOnly()`, I wouldn't be sad if we just overloaded `create()`, given the JavaDoc and the arguments make it pretty clear what we can possibly create. Not a big deal either way though...

Reply:
Well, I initially had them all as `create` and then changed them to this. I am more than happy to have them as all create overloads. I'll change it.